public/pages/DetectorDetail/containers/DetectorDetail.tsx

```diff
@@ -67,7 +69,7 @@ const tabs = [
   },
   {
     id: DETECTOR_DETAIL_TABS.CONFIGURATIONS,
-    name: 'Detector configuration',
+    name: 'Detector Configuration',
```

Comment:
`Detector configuration` is correct, we should change `Anomaly Results` to `Anomaly results`

Reply:
Oh I see, thanks. Will change.